TestProjects/UniversalGraphicsTest/Assets/Test/Editor/PreBuildStep.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+public class PreBuildStep : IPrebuildSetup
+{
+    List<string> ScenesToNotBuild = new List<string> {"035_Shader_TerrainShaders", };
```

Comment:
Do we also want 132_DetailMapping?

Reply:
Looking at the remaining failures in the tests that have run so far, it also looks like Linux_Vulkan_playmode_cache_mono_Linear for 035_Shader_TerrainShader is also still failing?

Reply:
Detail mapping seems to only be for Linux playmode as well. The change seems like it worked:
https://yamato-artifactviewer.prd.cds.internal.unity3d.com/16e172d9-58c4-4531-b316-5b618e0f6495%2FTest_report%2FUtrTestReport2%2FTestProjects%2FUniversalGraphicsTest%2Ftest-results/TestReportV1.html

Reply:
I had the 132_detailmapping in my commit, just forgot to push last night. doh! :)